shared/module/MakeShared/tests/Bootstrap.php

```diff
@@ -0,0 +1,5 @@
+<?php
```

Comment:
I think this will deal with loading files, so we shouldn't need the autoload.php file. I would try removing the latter and see if everything still works.

Reply:
Done